services/audio-broadcaster/templates/service.yaml

```diff
@@ -0,0 +1,20 @@
+{{- range $index, $mic := .Values.microphones }}
```

Comment:
Since you are using a loop, all references need to be changed to the following:
`.` -> `$`
`.Values` -> `$.Values`

Reply:
thanks for that!